trunk/SUAVE/Methods/Missions/Segments/Common/Energy.py

```diff
@@ -36,16 +37,66 @@ def initialize_battery(segment):
                                 
     """
     
+    if 'battery_age_in_days' not in segment:
```

Comment:
if this is something to look for it should be defaulted in the appropriate place. Otherwise a new user might never know this is an option

Reply:
addressed. Called battery age, and the unit is days.